emailtemplateservice/tool/src/main/java/org/sakaiproject/emailtemplateservice/config/ThymeleafConfig.java

```diff
@@ -0,0 +1,87 @@
+/******************************************************************************
+* Copyright (c) 2020 Apereo Foundation
```

Comment:
Have you used a Delorean to go back to 2020? Or maybe you used it to go to the future 😂😂

Reply:
jajajja